src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import command.*;
```

Comment:
Do not use wildcard imports. List all imports down individually.

Reply:
Ok!